### PR TITLE
Fix Cert main Domain and Move Volume Folders to .env

### PR DESCRIPTION
**Pull request by joshkreud**
- Cert Domain
- Volume folder "Mailu" in .env instead of compose.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -53,10 +53,10 @@ services:
     labels:
       - traefik.enable=true
       - traefik.docker.network=traefik
-      - traefik.http.routers.mailu.rule=Host(`mailu.${DOMAINNAME}`) || Host(`imap.${DOMAINNAME}`) || Host(`smtp.${DOMAINNAME}`)
+      - traefik.http.routers.mailu.rule=Host(`mailu.${DOMAINNAME}`)
```
Comment (hengeb) [inline docker-compose.yml:55]: why? imap and smtp subdomains have to be routed to mailu

File: .env.sample [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # edit and save as .env
 mailu_subnet=172.19.0.0/16
 DOMAINNAME=docker.localhost
-VOLUMES_DIR=./docker/volumes
+VOLUMES_DIR=./docker/volumes/mailu
```
Comment (hengeb) [inline .env.sample]: should not be changed here, because "." is the mailu stack folder

File: docker-compose.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ services:
       # Folder, which contains the acme.json
       - "${TRAEFIK_ACME_DIR}:/traefik"
       # Folder, where cert.pem and key.pem will be written
-      - "${VOLUMES_DIR}/mailu/certs:/output"
+      - "${VOLUMES_DIR}/certs:/output"
```
Comment (hengeb) [inline docker-compose.yml:30]: add empty volume folders in git?

File: docker-compose.yml (hunk 1 above)
Comment (joshkreud) [reply docker-compose.yml:55]: Traefik only routes port 80 and 443. Imap and smtp use different ports.

**Comment by joshkreud** [comment]
So PR is a lot cleaner now. 
Paths are now all concise, useless comments are gone.

**Review by hengeb — CHANGES_REQUESTED** [review]
instead of .gitkeep add a .gitignore with two lines:

    *
    !.gitignore

so the content that might be there in dev context will not make its way to the repo. 

create the other volume folders as well, not only the cert volume

**Comment by joshkreud** [comment]
Done.

**Review by hengeb — APPROVED** [review]

**Pull request merged**